packages/ckeditor5-dev-ci/bin/circle-workflow-notifier.js

```diff
@@ -97,7 +98,8 @@ async function getOtherJobsData() {
 function parseArguments( args ) {
 	const config = {
 		string: [
-			'task'
+			'task',
+			'ignore'
```

Comment:
A potential user can use:

* `--ignore job` to ignore a single job,
* `--ignore job1 --ignore job2` to ignore many jobs,
* `--ignore job1,job2` (as above, but isn't aware of _array_ syntax).

Let's check if the `--ignore` value is a string and normalize it as an array.

Also, make sure to update JSDoc.